src/client/ui/copy/copy.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'https://esm.sh/react@18.2.0'
+
+import { ENV } from '@/config'
+import { Message } from '../../shared/message.tsx'
+
+const { API_URL } = ENV
+
+export const Copy = ({ text }: { text: string }) => {
+  const [showCopied, setShowCopied] = React.useState(false)
+
+  const handleCopy = (text: string) => {
+    navigator.clipboard
+      .writeText(text)
+      .then(() => {
+        setShowCopied(true)
+        setTimeout(() => setShowCopied(false), 1000)
+        Message.show('Copied to clipboard', 'regular')
+      })
+      .catch(error => Message.show(<div>Failed to copy: {error.message}</div>, 'error'))
+  }
+
+  return (
+    <div className="copy__container  input_container">
+      <div className="input">{text}</div>
+      <div className="copy" onClick={() => handleCopy(text)}>
+        {showCopied ? (
+          <img src={`${API_URL}/img/success.svg`} alt="Success Icon" />
+        ) : (
+          <img className="copy__icon" src={`${API_URL}/img/copy.svg`} alt="Copy Icon" />
+        )}
```

Comment:
```suggestion
        {showCopied ? (
          <img src={`${API_URL}/img/success.svg`} alt="success" />
        ) : (
          <img className="copy__icon" src={`${API_URL}/img/copy.svg`} alt="copy" />
        )}
```
Для alt-ов используй краткие значения. Только представь как компонент перекособочит на устройстве с отключенными картинками!!!

И почему для одного img указан класс а для другого нет? Неужели только потому что у одного svg заданы подходящие размеры а у другого не нет? :)

Как вариант, запись можно сократить подобным образом:
```jsx
<img className="copy__icon" src={`${API_URL}/img/${showCopied ? 'success' : 'copy'}.svg`} alt="copy" />
```

Reply:
добавила класс, чтобы делать filter invert только для одной из иконок.

Reply:
Из названия класса это вообще не очевидно :)